WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionSheetViewController.swift

```diff
@@ -0,0 +1,278 @@
+import UIKit
```

Comment:
Is there a reason why we cannot reuse `ActionSheetViewController` like the create button does, instead of creating a whole new object that seems to do a very similar job?

Reply:
I was thinking of using `ActionSheetViewController` but it has a bit different behaviour and UI. To be able to use it we would need to refactor `ActionSheetViewController` to add ability to have different items/options, have option to change the spacing, etc. Basically I would need to make `ActionSheetViewController` a bit more generic.
I can look into it and maybe make a proposal on what would I refactor to be able to use `ActionSheetViewController` for sorting option. Should I do that?

Reply:
as discussed on slack, it's ok to use the new component.